UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs

```diff
@@ -120,6 +120,15 @@ private IEnumerator FlashEyesCoroutine(float flashTime)
 			lastFlashbangCoroutine = null;
 		}
 
+		public void Stop()
```

Comment:
Why flashes only?

There's like 5 other different effects that we have; and I'm pretty sure we already have a function somewhere that handles this.

Reply:
what other effects are there?

Reply:
Take a look at the camera prefab

Reply:
all the others are controlled by body synchronise stuff